sql/registrations_with_contacts.sql

```diff
@@ -71,7 +72,38 @@ LEFT JOIN (
         firstname IS NOT NULL AND
         lastname IS NOT NULL
       )
-      AS ownernames,
+    AS ownernames,
+
+    coalesce(
+      jsonb_agg(
+        distinct
+        jsonb_build_object(
+          'title', type, 
+          'value', (firstname || ' ' || lastname),
+          'address', nullif(concat_ws(' ', businesshousenumber, businessstreetname, businessapartment, businesszip), '')
```

Comment:
Do we think it's worth it to define this `address` prop as it's own object with keys for each part of the address? Perhaps we will need bits and pieces of the address handy in the future? The downside is that it adds unneeded complexity to the overall json, and will affect readability in the export CSV (although it's pretty bad to begin with...). What do folks think?

Reply:
interesting--regarding the export CSV, i wonder if we could join together all the fields into a string, analogous to the way we join some arrays of objects into comma-separated strings in the CSV export?  (except it'd be an object, not an array, we're joining, but same general concept)

Reply:
That said, overall I do like the idea of showing the individual fields.